.travis.yml

```diff
@@ -1,8 +1,22 @@
-  language: python
-  python:
-    - 3.5
-    - 3.6
-  install:
-    - make
-  script:
-    - make test
+language: python
+
+matrix:
+  include:
+    - os: linux
+    - os: osx
+      # workaround for https://github.com/travis-ci/travis-ci/issues/2312
+      language: generic
+      before_install:
```

Comment:
Does this mean that macOS builds may have a different version of Python to Linux builds, and importantly, different to what the documentation says is required?

Reply:
Good point, it seems to be 3.6 from the build console logs. I'll look for a way to stick to 3.5 there or to bump requirements (https://github.com/bamarni/dcos_cli/issues/2).